cmd/memcached_exporter/main.go

```diff
@@ -50,8 +59,30 @@ func main() {
 	level.Info(logger).Log("msg", "Starting memcached_exporter", "version", version.Info())
 	level.Info(logger).Log("msg", "Build context", "context", version.BuildContext())
 
+	if *serverName == "" {
+		*serverName = strings.Split(*address, ":")[0]
+	}
+	var (
+		tlsConfig *tls.Config
+		err       error
+	)
+	if *enableTls {
```

Comment:
Rather than have a separate boo flag, why not use `if *certfile != ""`.

It would also be good to validate the non-empty values for the other params.

Reply:
`if *certfile != ""` assumes we're using MTLS. Having `enableTls` enables TLS without having the client send certificates.

I can take a look at validating the other parameters, thanks.

Reply:
Ahh, right, that makes sense.